lesson-4/script.js

```diff
@@ -0,0 +1,137 @@
+  // Вивести на сторінку в один рядок через кому числа від 10 до 20.
+
+  let result = "";
+
+  for (let i = 10; i <= 20; i++) {
+    if (i === 20) {
+      result = `${result + i}`;
+    } else {
+      result = `${result + i}, `
+    }
+  }
+  console.log(result);
+
+  // Вивести квадрати чисел від 10 до 20.
+
+  let numberSquare = '';
+
+  for (let i = 10; i <= 20; i++) {
+    let square = Math.pow(i, 2);
+    if (square === 400) {
+      numberSquare += square;
+    } else {
+      numberSquare += square + ', ';
+    }
+  }
+  console.log(numberSquare);
+
+  // Вивести таблицю множення на 7.
+
+  for (let i = 1; i <= 10; i++) {
+    let c1 = `${i}` * 7;
+    let result = `${i} * 7 = ${c1}`;
+    console.log(result);
+  }
+
+  // Знайти суму всіх цілих чисел від 1 до 15.
+
+  let sumNumberByOneToFifteen = 1;
+
+  for (let i = 1; i <= 15; i++) {
+    sumNumberByOneToFifteen += i;
+  }
+  console.log(sumNumberByOneToFifteen);
+
+  let multipleNumberByFifteenToThirtyfive = 1;
+
+  for (let i = 15; i <= 35; i++) {
+    multipleNumberByFifteenToThirtyfive *= i;
+  }
+
+  console.log(multipleNumberByFifteenToThirtyfive);
+
+  // Знайти середнє арифметичне всіх цілих чисел від 1 до 500.
+
+  let avarage = 0;
```

Comment:
typo in variable name (should be average)
```suggestion
  let average = 0;
```